functions/scraper/func.py

```diff
@@ -0,0 +1,60 @@
+import ujson
+import os
+import flickrapi
+import random
+import requests
+import fdk
+
+from cloudevents.sdk import converters
+from cloudevents.sdk import marshaller
+from cloudevents.sdk.converters import structured
```

Comment:
Consider using binary format instead of structured.

Reply:
why?

Reply:
simpler, less efforts to parse the event itself.